packages/ruby_slippers/docs/README.md

```diff
@@ -0,0 +1,48 @@
+# *NOTICE:* This app is in its _experimental_ phase and a stable version has not yet been released
+
+# Ruby Slippers 👠👠
+
+_There's no place like the general layout and the top menu for which the home button lives._
+
+At [Power](https://powerhrg.com/#!/career), we have a suite of applications that provide a wealth of functionality to our users, many of which were broken out from our monolith. We want all of these applications to have the same look and feel and cohesive experience. While we rely on [playbook](https://playbook.powerapp.cloud/) for bootstrap-like functionality, we want to put some of these pieces and other things together to give us a general layout and functionality for a top menu bar and other items that are likely to be consistently shown across every page. We also want to provide some injectable containers so that applications can magically add their own functionality into the menu if they would like to do so.
```

Comment:
```suggestion
At [Power](https://techatpower.com), we have a suite of applications that provide a wealth of functionality to our users, many of which were broken out from our monolith. We want all of these applications to have the same look and feel and cohesive experience. While we rely on [playbook](https://playbook.powerapp.cloud/) for bootstrap-like functionality, we want to put some of these pieces and other things together to give us a general layout and functionality for a top menu bar and other items that are likely to be consistently shown across every page. We also want to provide some injectable containers so that applications can magically add their own functionality into the menu if they would like to do so.
```